build.gradle.kts

```diff
@@ -68,3 +69,23 @@ tasks.named<DependencyUpdatesTask>("dependencyUpdates").configure {
         isNonStable(candidate.version)
     }
 }
+
+forbiddenApis {
+    bundledSignatures = setOf("jdk-deprecated", "jdk-non-portable")
```

Comment:
Let's try to use all bundled signatures https://github.com/policeman-tools/forbidden-apis/wiki/BundledSignatures

Reply:
When adding build signatures such as Unsafe or Refelection in existing changes, errors occurred in existing code such as String.format();
After searching about this, I found this issue https://github.com/policeman-tools/forbidden-apis/issues/119
I'm not sure if I understand this issue correctly.
Based on the four build signatures in the example in the issue of this pull request, I added only two build signatures that did not cause errors in the existing code.

How should we proceed?

Reply:
> How should we proceed?

Use
```
bundledSignatures = setOf("jdk-unsafe", "jdk-deprecated", "jdk-internal", "jdk-non-portable", "jdk-system-out", "jdk-reflection")
```
and rewrite `String.format` usages